app/views/devise/passwords/new.html.slim

```diff
@@ -0,0 +1,25 @@
+.page-details--creme-bg
+  .container
+    = render 'shared/alerts'
+    .row
+      .col-xs-12.col-md-8.col-md-offset-2.col-lg-6.col-lg-offset-3
+        .form-box.form-box--middle
+          .form-box__title
+            h2 Forgot your password?
+          .form-box__body
+            = form_for(resource, as: resource_name, url: password_path(resource_name), html: { class: 'form-horizontal', method: :post }) do |f|
```

Comment:
@ptrikutam Did you restart your server after switching branches? Could you try? Because I just solved the issue like that in local - probably Devise doing some stuff during the app boot up.